scripts/run_commands

```diff
@@ -35,26 +39,30 @@ conda config --set show_channel_urls True
 conda config --set channel_priority strict
 conda config ${additional_channel} --add channels conda-forge
 conda config --show-sources
-conda update --all --yes
-conda clean -tipy
 
-# Install conda build and deployment tools.
-conda install --yes --quiet \
-    git patch \
-    python=3.9 pip \
-    mamba boa \
-    conda-build anaconda-client
+# Update everything first
+mamba update --yes --all
```

Comment:
This seems to be updating conda to 22.X which is breaking things with mamba 0.24 not fun.

I've been trying to update the versions we have in miniforge, but I can't seem to installa mamba, conda, and boa for all supported platforms: https://github.com/conda-forge/miniforge/pull/358

Reply:
Do `mamba` or `boa` need a repodata patch? It looks like it, because the solver shouldn't take us to a situation where something doesn't work :/

Reply:
It seems that a new boa got released hours ago.

Reply:
Oh, hopefully that fixes things!

Reply:
(Or is that what broke things?)

Reply:
Ultimately we can pin conda to 4.14 if needed via `CONDA_PINNED_PACKAGES=conda=4.14`

Reply:
It looks like only mamba 0.25.0 [was patched]( https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/322#issuecomment-1260553604 ), but mamba 0.24.0 is shown above. Do we need to ...?

* Patch older mamba's
* Update the installer used in the images (and/or the installer itself) to a newer mamba

Reply:
Looks like there is a new Mambaforge release (thanks Mark! 🙏). Submitted draft PR ( https://github.com/conda-forge/docker-images/pull/222 ) to update. Happy to change to ready for review once the Mambaforge release is complete and we are ready to update here.

Reply:
This should now be resolved with PR ( https://github.com/conda-forge/docker-images/pull/224 )